website/templates/project/view_file.mako

```diff
@@ -28,15 +28,17 @@
           <ol class="breadcrumb">
             <li><a href="{{ node.urls.files }}" data-bind="text: node.title"></a></li>
             <li class="active overflow" data-bind="text: file.provider"></li>
-            <li class="active overflow" data-bind="text: file.name"></li>
+            <!-- ko foreach: file.path.split('/').slice(1) -->
+            <li class="active overflow" data-bind="text: $data"></li>
+            <!-- /ko -->
           </ol>
 
-          <a class="btn btn-success btn-md" href="{{ currentVersion().osfDownloadUrl }}" data-bind="click: currentVersion().download">
+          <a class="btn btn-success btn-md fileDownload" href="{{ currentVersion().osfDownloadUrl }}" data-bind="click: currentVersion().download">
```

Comment:
> Use .lowercase-and-dashes for class names and #camelCase for IDs.

http://cosdev.readthedocs.org/en/latest/style_guides/html_css.html

Also, we should namespace this better to avoid inadvertently hiding elements. Something like: `osf-hideable-download-btn`.